airflow/dags/sprint_3/msi_7_dag.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+
+import pendulum
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import (
+    PythonOperator,
+    BranchPythonOperator,
+)
+from airflow.utils.trigger_rule import TriggerRule
+
+import scripts.credentials as cr
+import scripts.main as m
+
+restricted_dates = [
+    datetime(2024, 6, 11).strftime("%Y-%m-%d"),
+    datetime(2024, 6, 13).strftime("%Y-%m-%d"),
+    datetime(2024, 6, 15).strftime("%Y-%m-%d"),
+    datetime(2024, 6, 25).strftime("%Y-%m-%d"),
+]
+
+
+def _skip_messages_on_dates():
+    date_now = datetime.today().strftime("%Y-%m-%d")

```

Comment:
Better to use Airflow related variables, not datetime.today() - what if you would relaunch some processes in the past?